db/migrate/20160816144624_create_damages.rb

```diff
@@ -0,0 +1,14 @@
+class CreateDamages < ActiveRecord::Migration
+  def change
+    create_table :damages do |t|
+      t.string :uuid
+      t.string :damage_issued_reservation_uuid
```

Comment:
Personally I would prefer to omit the `uuid` from these column names, since we already know that UUIDs are the only way to refer inter-app to items.